src/qml/QFieldCloudPopup.qml

```diff
@@ -16,8 +16,7 @@ Popup {
     header: PageHeader {
       title: qsTr('QFieldCloud')
 
-      showBackButton: cloudProjectsModel.currentProjectData.Status !== QFieldCloudProjectsModel.Uploading
-                        || cloudProjectsModel.currentProjectData.Status !== QFieldCloudProjectsModel.Downloading
```

Comment:
Why you removed this part?

Reply:
I guess you made the back button always visible since you prevented the user to edit

Reply:
That's correct -- also, the condition was broken to begin with :) it should have been !download && !upload :)

Reply:
clear, thanks